editor/src/components/canvas/canvas-context-menu.spec.browser2.tsx

```diff
@@ -1060,37 +1062,81 @@ describe('canvas context menu', () => {
 
       await renderResult.dispatch(selectComponents([testValuePath], false), true)
 
-      // Wrap it in a div.
+      // Wrap it in a Group.
       const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
       const element = renderResult.renderedDOM.getByTestId('target-div')
       const elementBounds = element.getBoundingClientRect()
       await openContextMenuAndClickOnItem(
         renderResult,
         canvasControlsLayer,
         elementBounds,
-        'Wrap in div',
+        'Group Selection',
       )
       await renderResult.getDispatchFollowUpActionsFinished()
 
       expect(getPrintedUiJsCodeWithoutUIDs(renderResult.getEditorState())).toEqual(
         makeTestProjectCodeWithSnippetWithoutUIDs(
           `<div style={{ ...props.style }}>
-             <div style={{ position: 'absolute'}}>
+             <Group style={{ position: 'absolute', left: 154, top: 134, width: 150, height: 150 }}>
                <div
                  style={{
                    height: 150,
                    width: 150,
                    position: 'absolute',
-                   left: 154,
-                   top: 134,
+                   left: 0,
+                   top: 0,
                    backgroundColor: 'lightblue',
                  }}
                  data-testid='target-div'
                />
-             </div>
+             </Group>
            </div>`,
         ),
       )
     })
+
+    it('wrap in Group doesnt yet work for non position: absolute elements', async () => {
+      const renderResult = await renderTestEditorWithCode(
+        makeTestProjectCodeWithSnippet(
+          `<div style={{ ...props.style }} data-uid='aaa'>
+             <div
+               style={{
+                 height: 150,
+                 width: 150,
+                 left: 154,
+                 top: 134,
+                 backgroundColor: 'lightblue',
+               }}
+               data-uid='target-div'
+               data-testid='target-div'
+             />
+           </div>`,
+        ),
+        'await-first-dom-report',
+      )
+
+      const testValuePath = EP.fromString(
+        `${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:aaa/target-div`,
+      )
+
+      await renderResult.dispatch(selectComponents([testValuePath], false), true)
+
+      // Wrap it in a Group.
+      const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+      const element = renderResult.renderedDOM.getByTestId('target-div')
+      const elementBounds = element.getBoundingClientRect()
+      await openContextMenuAndClickOnItem(
+        renderResult,
+        canvasControlsLayer,
+        elementBounds,
+        'Group Selection',
+      )
+      await renderResult.getDispatchFollowUpActionsFinished()
+
+      expect(renderResult.getEditorState().editor.toasts.length).toBe(1)
```

Comment:
Minor: Surely just checking that the array is equal to an array containing the text is simpler?